pfs/lineLists/ArCdHgKrNeXe.txt

```diff
@@ -281,7 +282,7 @@
  612.496        3            HgI   0
  623.608       50            HgI   0
  670.35         2            HgI   0
- 671.819      600            HgI   3
+ 671.819      600            HgI   0
```

Comment:
So this line has been promoted from unreliable to OK? Why?

Reply:
The commit message explains:
> The previous line list vetting I did used sims data, which I
> believe only used Neon, not Hg/Ar/Kr. Some lines that were
> flagged are good lines, and useful for calibrating HgAr arcs.

Reply:
Ok thanks.